src/components/Events/Events.js

```diff
@@ -64,7 +64,7 @@ function Events() {
             {EventData.filter(edge => {
               return stringToDate(edge.node.date) > new Date();
             }).map((edge, i) => {
-              return i < maxEvents && <EventItem {...edge.node} />;
+              return i < maxEvents && <EventItem {...edge.node} key={edge.node.title} />;
```

Comment:
Are there any other unique identifiers that can be used?

Reply:
using i is much smarter! Just changed it